PWGMM/Mult/Tasks/dndeta-mft.cxx

```diff
@@ -92,6 +93,7 @@ struct PseudorapidityDensityMFT {
 
   void init(InitContext&)
   {
+    ambTrackIds.clear();
```

Comment:
This needs to be done for each dataframe, so in `process`, not in init

Reply:
ok ! thx